.github/workflows/tf-merge.yml

```diff
@@ -0,0 +1,140 @@
+name: "Merge to Master"
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - "terraform/**"
+defaults:
+  run:
+    working-directory: terraform/deploy/
+
+jobs:
+  management-dev:
+    name: "Terraform Apply Mangement-Dev"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: hashicorp/setup-terraform@v1
+        with:
+          terraform_version: 0.12.19
+      - uses: actions/setup-python@v2
+        with:
+          python-version: "3.x"
+
+      - name: "Install dependencies"
+        id: deps
+        run: pip install Jinja2 PyYAML boto3
+
+      - name: "Bootstrap"
+        id: bootstrap
+        run: |
+          python bootstrap_terraform.py
+          sed -i '/^assume_role/ d' terraform/deploy/terraform.tfvars
+        working-directory: .
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.ACTIONS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.ACTIONS_SECRET_ACCESS_KEY }}
+          AWS_SECRETS_ROLE: ${{ secrets.AWS_GHA_ROLE_MGMT_DEV }}
+          AWS_DEFAULT_REGION: eu-west-2
+
+      - name: "Terraform Format"
+        id: fmt
+        run: terraform fmt
```

Comment:
I'm not entirely sure why fmt is being run, but if needed then it should have --recursive